### ISSUE TITLE
KlDivBackward

### ISSUE THREAD

**Issue by alshahrani2030** [huggingface/transformers#4268]
Hi,
I fine-tune bert and using KLDivLoss instead of CrossEntropyLoss in binary classification. when I call 
loss.backward()
I got this error !?

![error-KLDiv](https://user-images.githubusercontent.com/55197626/81508647-4dd85780-92d3-11ea-9547-a9690b733a61.PNG)

Thank you in advance
